widgetastic_manageiq/__init__.py

```diff
@@ -5017,17 +5017,20 @@ class MigrationProgressBar(Widget):
 
     ITEM_LOCATOR = './/div[contains(@class,"card-pf-match-height")]'
     TITLE_LOCATOR = './/div[h3[contains(@class,"card-pf-title")]]'
-    TIMER_LOCATOR = './/div[contains(@class,"active-migration-elapsed-time")]'
+    TIMER_LOCATOR = './div/div[contains(@class,"active-migration-elapsed-time")]'
     SIZE_LOCATOR = './/strong[contains(@id,"size-migrated")]'
     VMS_LOCATOR = './/strong[contains(@id,"vms-migrated")]'
     SPINNER_LOCATOR = './/div[contains(@class,"spinner")]'
     ERROR_LOCATOR = './/div/span[contains(@class,"pficon-error-circle-o")]'
     PROGRESS_BARS = './/div[@class="progress-bar"]'
     PROGRESS_DESCRIPTION = './/div[contains(@class,"progress-description")]'
+    locator = './/div[contains(@class,"migrations")]/div/div/div/div/div'
 
-    def __init__(self, parent, locator, logger=None):
+    def __init__(self, parent, logger=None):
         Widget.__init__(self, parent, logger=logger)
-        self.locator = locator
+
+    def __locator__(self):
```

Comment:
I think you should just use `ROOT` attribute here for this, instead of this boilerplate to create `__locator__` method.

https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/widget/base.py#L246